manifests/integrations/eventhub-credentials-sync/_base/sync.yaml

```diff
@@ -71,16 +71,16 @@ spec:
                     | tr [ %
                   )"
                 # shellcheck disable=SC2086
-                if [ "${job_ids}" ]; then
-                  kill ${job_ids}
+                if [ "$job_ids" ]; then
+                  kill $job_ids
                 fi
                 wait
                 echo "Graceful exit -- $(date)"
               }
 
               trap graceful_exit INT TERM
 
-              echo "Loop started (period: ${SYNC_PERIOD} s) -- $(date)"
+              echo "Loop started (period: ${SYNC_PERIOD:-3600} s) -- $(date)"
```

Comment:
this is still getting substituted preventing the intended effect of the configmap overriding the value, you can set it via post build if you want otherwise any suggestions for fixing it properly?

(I personally don't need to override this)